TypeScript types are broken after last updates

Compiling with TypeScript is actually not possible
```
Using tsc v2.4.2
node_modules/react-player/index.d.ts(32,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(32,5): error TS1131: Property or signature expected.
node_modules/react-player/index.d.ts(39,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(39,5): error TS1131: Property or signature expected.
node_modules/react-player/index.d.ts(46,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(46,5): error TS1131: Property or signature expected.
node_modules/react-player/index.d.ts(49,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(49,5): error TS1131: Property or signature expected.
```
PR follows

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,11 +1,11 @@
 import * as React from 'react';
 
-export interface SourceSpec {
+export interface SourceProps {
   src: string;
   type: string;
 }
 
-export interface TrackSpec {
+export interface TrackProps {
   kind: string;
   src: string;
   srcLang: string;
@@ -13,7 +13,7 @@ export interface TrackSpec {
 }
 
 export interface ReactPlayerProps {
-  url?: string|string[]|SourceSpec[];
+  url?: string|string[]|SourceProps[];
   playing?: boolean;
   loop?: boolean;
   controls?: boolean;
@@ -51,7 +51,7 @@ export interface ReactPlayerProps {
   };
   fileConfig?: {
     attributes: Object;
-    tracks: TrackSpec[];
+    tracks: TrackProps[];
     forceAudio: boolean;
     forceHLS: boolean;
     forceDASH: boolean;

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,7 +1,19 @@
 import * as React from 'react';
 
+export interface SourceSpec {
+  src: string;
+  type: string;
+}
+
+export interface TrackSpec {
+  kind: string;
+  src: string;
+  srcLang: string;
+  default?: boolean;
+}
+
 export interface ReactPlayerProps {
-  url?: string|array;
+  url?: string|string[]|SourceSpec[];
   playing?: boolean;
   loop?: boolean;
   controls?: boolean;
@@ -16,37 +28,37 @@ export interface ReactPlayerProps {
   hidden?: boolean;
   className?: string;
   soundcloudConfig?: {
-    clientId: string,
-    showArtwork: boolean
+    clientId: string;
+    showArtwork: boolean;
   };
   youtubeConfig?: {
-    playerVars: Object,
-    preload: boolean
+    playerVars: Object;
+    preload: boolean;
   };
   facebookConfig?: {
-    appId: string
+    appId: string;
   };
   dailymotionConfig?: {
-    params: Object,
-    preload: boolean
-  }),
+    params: Object;
+    preload: boolean;
+  };
   vimeoConfig?: {
-    iframeParams: Object,
-    preload: boolean
+    iframeParams: Object;
+    preload: boolean;
   };
   vidmeConfig?: {
-    format: string
-  }),
+    format: string;
+  };
   fileConfig?: {
-    attributes: Object,
-    tracks: array,
-    forceAudio: boolean,
-    forceHLS: boolean,
-    forceDASH: boolean
-  }),
+    attributes: Object;
+    tracks: TrackSpec[];
+    forceAudio: boolean;
+    forceHLS: boolean;
+    forceDASH: boolean;
+  };
   wistiaConfig?: {
-    options: Object
-  }),
+    options: Object;
+  };
   onReady?(): void;
   onStart?(): void;
   onPlay?(): void;
@@ -59,6 +71,6 @@ export interface ReactPlayerProps {
   onProgress?(state: { played: number, loaded: number }): void;
 }
 
-export default class ReactPlayer extends React.Component<ReactPlayerProps, any>{
+export default class ReactPlayer extends React.Component<ReactPlayerProps, any> {
   seekTo(fraction: number): void;
 }

